socketio/src/packet.rs

```diff
@@ -117,6 +123,13 @@ impl From<&Packet> for Bytes {
     }
 }
 
+impl TryFrom<Bytes> for Packet {
+    type Error = Error;
+    fn try_from(value: Bytes) -> Result<Self> {
+        Packet::try_from(&value)
+    }
+}
```

Comment:
That's nice to have!

Reply:
The existing choice to use a reference (as it doesn't require ownership) was fascinating, don't see that very often but makes sense!